docs/blocks-storage/querier.md

```diff
@@ -878,4 +878,9 @@ blocks_storage:
     # will be stored. 0 or less means disabled.
     # CLI flag: -blocks-storage.tsdb.max-exemplars
     [max_exemplars: <int> | default = 0]
+
+    # [EXPERIMENTAL] Configures the maximum capacity for out-of-order chunks (in
+    # samples). If set to <=0, default value 32 is assumed.
+    # CLI flag: -blocks-storage.tsdb.out-of-order-cap-max
+    [out_of_order_cap_max: <int> | default = 32]
```

Comment:
It feels weird to me. Shall we extract TSDB configs from block storage section? Having them in querier and SG is werid.

Reply:
I think querier comonent doesn't carray about this field,  Is this due to the use of automatic template document?

Reply:
Yeah I think querier and store gateway uses some configs from block storage.
But tsdb configs shouldn't be related.

Reply:
If out of order is enabled, users should not care about the configuration here (the burden of configuration is too heavy), unless there is a maximum capacity limit (the default is the largest).

Reply:
If you are talking about the ooo capacity, I don't think this change would introduce additional burden to them.

This is a global value, not per tenant.